Code/Tools/Libs/ToolsFoundation/FileSystem/Implementation/FileSystemModel.cpp

```diff
@@ -490,20 +531,24 @@ ezResult ezFileSystemModel::ReadDocument(ezStringView sAbsolutePath, const ezDel
   if (!m_bInitialized)
     return EZ_FAILURE;
 
+  ezStringBuilder sAbsolutePath2(sAbsolutePath);
+  sAbsolutePath2.MakeCleanPath();
+  sAbsolutePath2.TrimWordEnd("/");
```

Comment:
I wouldn't use this, but just Trim. TrimWord only removes one instance IIRC.

Reply:
Oh good point!